app/Exceptions/ProjectNotFoundException.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Exceptions;
+
+use Exception;
+use Throwable;
+
+class ProjectNotFoundException extends Exception
+{
+    public const MESSAGE = 'No s\'ha trobat cap projecte associat a aquest estudiant amb ID: %s';
+
+    public function __construct($studentId, $code = 0, Throwable $previous = null)
```

Comment:
Following Jordi's recommendation in other custom exception files, I suggest replacing ``$code = 0`` in the constructor with ``$code = 404``.

This ``$code = 404`` will be used later in ``StudentProjectsDetailController.php`` to replace the respective hard-coded 404 in each corresponding catch.

**This change has already been applied to the rest of the custom exceptions that are already merged into develop**.